lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -48,7 +75,7 @@
     <button class="action action-cancel">
         <%=
             interpolate_text(
-                _.escape(gettext("Cancel {span_start}a new team{span_end}")),
+                _.escape(gettext("Cancel {span_start} team {span_end}")),
```

Comment:
If its a product requirement, perhaps we should carefully re-consider it.  Jakob Nielson questioned the efficacy of the Cancel button back in 2000:
http://www.nngroup.com/articles/reset-and-cancel-buttons/

We have to ask ourselves if its necessary.  99 times out of 100 I find it is not.